usaspending_api/common/elasticsearch/json_helpers.py

```diff
@@ -13,4 +14,20 @@ def json_str_to_dict(string: str) -> dict:
     try:
         return json.loads(string.encode("unicode_escape"))
     except json.decoder.JSONDecodeError as e:
-        raise e
+
+        # Try to parse the string with Regex before throwing error
+        key_count_regex = r"\"[^\"]*\"\s?:"
+        grouping_regex = r"\"([^\"]*)\"\s?:\s?\"([^\"]*(?:(?:\w|\s)?(?:\"|\')?(?:\w|\s)?)*[^\"]*)(?:(?:\"\,)|(?:\"\}))"
+
+        key_count_matches = re.findall(key_count_regex, string)
+        grouping_matches = re.findall(grouping_regex, string)
+
+        # Need to verify the correct number of elements in case grouping regex didn't work
+        if (
+            isinstance(key_count_matches, list)
+            and isinstance(grouping_matches, list)
+            and len(key_count_matches) == len(grouping_matches)
+        ):
+            return {key: value for key, value in grouping_matches}
+        else:
+            return e  # Return the JSONDecodeError since we couldn't salvage with regex
```

Comment:
At a minimum, the function needs to raise the exception instead of returning it.

Would be nice to include more details if it fails

```suggestion
            raise json.decoder.JSONDecodeError(f"Unable to parse '{string}' even using regex")
```

Reply:
Oh geez... that is what I get for trying to make a quick change.

Reply:
The quicker the change, the longer it takes